src/scripts/create-keystore-properties.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PARAM_RELEASE_KEY_ALIAS=$(eval echo "\$$PARAM_RELEASE_KEY_ALIAS")
+
+PARAM_RELEASE_KEY_PASSWORD=$(eval echo "\$$PARAM_RELEASE_KEY_PASSWORD")
+
+PARAM_RELEASE_STORE_PASSWORD=$(eval echo "\$$PARAM_RELEASE_STORE_PASSWORD")
```

Comment:
Normally would group these evals together, but nbd

Reply:
You might not even need the `eval` here. Something like this would suffice:

```
PARAM_RELEASE_KEY_ALIAS="${!PARAM_RELEASE_KEY_ALIAS}"
```

Example: https://github.com/game-ci/unity-orb/blob/main/src/scripts/prepare-env.sh#L14

I would, however, resolve the value from `PARAM_RELEASE_KEY_ALIAS` to a variable with a different name. Always good to try and keep your variables immutable. You can even enforce this in BASH with the `readonly` keyword.

Reply:
The above ^ assumes that the value in the variables you are evaluating doesn't contain variables that should be resolved inside them. Otherwise, eval really is the way to go.

Reply:
@EricRibeiro I tried the approach you gave but I was having issues so I instead made the values immutable and kept the eval

Reply:
eval is necessary for reducing a string with one or more EVs in it. The ! resolves the indirect EV so the command that uses it gets the nested value.

Reply:
I might be missing something, but if `PARAM_RELEASE_KEY_ALIAS` holds a value with the `env_var_name` type of parameter, the exclamation alone should work. Here is a simulation of the variable expansion process: http://tpcg.io/_IH691P.

@ryanbourdais, what issues were you having?

Reply:
@EricRibeiro the issue was when fastlane went to deploy it tried using ********* as credentials as that was what was written to the keystore.props file because they are env variables